apps/storefront/src/components/RegisteredCloseButton.tsx

```diff
@@ -1,5 +1,6 @@
 import { Dispatch, SetStateAction, useContext } from 'react';
 import { useNavigate } from 'react-router-dom';
+import { useB3Lang } from '@b3/lang';
```

Comment:
This import is not necessary

Reply:
noticed this, but was afk, will fix in Monday

Reply:
fixed